tests/common.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(unused_macros)]
+#![allow(unused)]
```

Comment:
This is dangerous. I think it would be better to no have it at all, but if it's really needed, then:
```suggestion
#![allow(dead_code)]
```